bid.proto

```diff
@@ -0,0 +1,61 @@
+syntax = "proto3";
+package rusk;
+
+import "basic_fields.proto";
+import "transaction.proto";
+import "keys.proto";
+
+message Bid {
+    PoseidonCypher encrypted_data = 1;
+    BlsScalar nonce = 2;
+    BlsScalar hashed_secret_k = 3;
+    JubJubCompressed R = 4;
+    JubJubCompressed pk = 5;
+    JubJubCompressed c = 6;
+}
+
+// We will need to send a secret to encrypt the value, blinder
+// & secret
+message BidTransactionRequest {
+    // TODO: Review type
+    bytes K = 1;
+    BlsScalar bid_value = 2;
+    JubJubCompressed pk = 3;
+    JubJubCompressed R = 4;
+    BlsScalar seed = 5;
+    fixed64 expiration_ts = 6;
+    uint32 elegibility_ts = 7;
+    // The call will edit the Tx `calldata` field with the Bid fn call
+    Transaction tx = 8;
+}
+
+message BidTransactionResponse {
+    Bid bid = 1;
+    BlsScalar bid_hash = 2;
```

Comment:
The bid hash is part of the response because we assume the receiver won't be able to calculate the hash of the bid? Considering the hash is calculated only with public data.

Reply:
He will not be able ye. Since go has no impl for sponge hash.